### PR TITLE
[DM-32254] Use PixelizationABC as the base for Python pixelization classes

### PR DESCRIPTION
**Pull request by tgoldina**
Creating PR to trigger tests using git+git://github.com/lsst/sphgeom@tickets/DM-32254#egg=lsst_sphgeom as a dependency.

## Checklist

- [ ] ran Jenkins
- [ ] added a release note for user-visible changes to `doc/changes`

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ pyyaml >= 5.1
 astropy >= 4.0
 click >7.0
 sqlalchemy >= 1.3
-git+git://github.com/lsst/sphgeom@master#egg=lsst_sphgeom
+git+git://github.com/lsst/sphgeom@tickets/DM-32254#egg=lsst_sphgeom
```
Comment (timj) [inline requirements.txt]: Please remember to remove this before merging.

**Review by timj — APPROVED** [review]
Thanks for doing this.  Looks great.

**Comment by timj** [comment]
@tgoldina  can you delete both the commits that fiddle with the sphgeom branch? They don't need to be in the history at all.

**Comment by timj** [comment]
(I guess this means squashing the two commits together)

**Comment by tgoldina** [comment]
@timj Sorry, I missed your suggestion about squashing two commits into one. Will do next time.

**Pull request merged**